### DISCUSSION TITLE
Hello

### DISCUSSION THREAD

**Discussion by dolA-0** [yagiziskirik/Kerbal-Telemetry#31] [Q&A] [answered]
The mod is very nice, but maybe because I use more than 70 mods along with RSS/RO, the telemetry never once drew the graphs that should have been on the second page...

**Comment by yagiziskirik** [comment, marked as answer]
Hi @dolA-0!

I am aware of the situation, but couldn't create free time to fix the issue. As soon as I can get back to the project, I will let you know. I am so sorry for not solving the issue at the moment.

**Reply by dolA-0** [reply]
Oh no, everything is fine. This is a really great modification, with its help it is very interesting to watch the game process. Like here I have for example ![Screenshot_2024-04-26-22-27-19-647_com.google.android.youtube.jpg](https://github.com/yagiziskirik/Kerbal-Telemetry/assets/150825869/1b434bc2-a694-4a04-9189-46ef5ee0f8e7)

**Comment by yagiziskirik** [comment]
That looks awesome! We can do it for the in-game screen as well as the addon itself. I would really like to hear your ideas once I get back to the project if you are willing to share them as well.

**Reply by dolA-0** [reply]
Well, here you can come up with a lot of ideas. To make it look like my previous image, I just added some web pages to the OBS, and reacted so that only the necessary information was visible (tried to reproduce the SpaceX overlay). As for ideas, I don't know if it's feasible, but it would be really cool if you could get telemetry from multiple ships at the same time. As for the in-game overlay, I think it's a great idea that can replace the existing overlay, however, in order to use it, you need to be sure that everything will go according to plan, for example, using kOS scripts. I will be happy to help in any way I can.

**Reply by yagiziskirik** [reply]
Theoretically, it should be possible to get telemetry data from multiple ships, but I should try it out first. The in-game overlay replacement is a bit tricky, but everything is possible if you work hard enough. I don't know if I can offer responsive enough telemetry data without the game lagging, but I quite liked the idea and would definitely try that. Thanks for the ideas and I will announce it once I get back to it!
